appservice/src/intTest/groovy/nu/yona/server/BasicBuddyTest.groovy

```diff
@@ -382,6 +380,46 @@ class BasicBuddyTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(bob)
 	}
 
+	def 'Goal conflict of Richard with an 2k long URL is reported to Richard and Bob'()
```

Comment:
The post gives an advice to stay under 2000 characters because of search engine preferences and old IE versions. However, all modern browsers seem to support 8192 or more characters. So I would like that it is truncated.